Reliability/file_check_reliability.rb

```diff
@@ -0,0 +1,213 @@
+# Updated 4.24.20

```

Comment:
Would be useful to have a description of what this script does—even if it's specific to a particular video-coding scheme it helps with understanding how to adapt it for a different scheme.